### PR TITLE
Moved utilization reports

### PR DESCRIPTION
**Pull request by tbaxter-18f**
Moves utilization reports from behind a menu only superusers can see and into a location where they can be viewed by anyone. Transparency!

### CODE REVIEW

**Comment by amymok** [comment]
@tbaxter-18f Can you update the PR description above: https://github.com/18F/tock/pull/966#issue-372537668

File: tock/tock/templates/_navigation.html [hunk 1]
```diff
@@ -10,11 +10,24 @@
           <span>Projects</span>
         </a>
       </li>
-      <li>
-        <a href="{% url 'reports:ListReports' %}" class="usa-nav-link">
-          <span>Reports</span>
-        </a>
-      </li>
+      {% if user.is_staff %}
```
Comment (amymok) [inline tock/tock/templates/_navigation.html]: This should be `request.user.is_staff`

File: tock/tock/templates/_navigation.html [hunk 2]
```diff
@@ -10,11 +10,24 @@
...
+        <li>
```
Comment (amymok) [inline tock/tock/templates/_navigation.html]: This doesn't render for me.  No menu actually shows up.

File: tock/tock/templates/_navigation.html (hunk 1 above)
Comment (tbaxter-18f) [reply tock/tock/templates/_navigation.html]: Huh. I thought we passed user into context, but it appears you're right!

File: tock/tock/templates/_navigation.html (hunk 2 above)
Comment (tbaxter-18f) [reply tock/tock/templates/_navigation.html]: We may need to be sure you're set up as superuser and staff in the admin.

File: tock/tock/templates/_navigation.html [hunk 3]
```diff
@@ -10,11 +10,24 @@
...
       </li>
-      <li>
-        <a href="{% url 'reports:ListReports' %}" class="usa-nav-link">
-          <span>Reports</span>
-        </a>
-      </li>
+      {% if request.user.is_staff %}
+        <li>
+          <button class="
+            usa-accordion-button usa-nav-link" aria-expanded="false" aria-controls="side-nav-2">
```
Comment (amymok) [inline tock/tock/templates/_navigation.html]: The `side-nav-2` is conflicting with the one under Administrative, this is causing the submenu not rendering.  So choose something else here.

File: tock/tock/templates/_navigation.html [hunk 4]
```diff
@@ -10,11 +10,24 @@
...
+            <span>Reports</span>
+          </button>
+          <ul id="side-nav-2" class="usa-nav-submenu" aria-hidden="true">
```
Comment (amymok) [inline tock/tock/templates/_navigation.html]: Same comment as above for the `side-nav-2`.  Choose another id.

Comment (tbaxter-18f) [reply tock/tock/templates/_navigation.html]: Good catch, Amy. Those are also bad IDs, so I've made them a bit more descriptive.

Comment (amymok) [reply tock/tock/templates/_navigation.html]: @tbaxter-18f Did you push the commit?

Comment (tbaxter-18f) [reply tock/tock/templates/_navigation.html]: I did now :-)

**Comment by amymok** [comment]
There's no changes in dependency, Snyk is being funky here for pending.  Will bypass and merge.

**Pull request merged**